yearn/prices/magic.py

```diff
@@ -64,14 +64,16 @@ def find_price(
 
     price = None
     if token in constants.stablecoins:
-        if chainlink and token in chainlink:
+        if chainlink and token in chainlink and block >= contract_creation_block(chainlink.get_feed(token).address):
             price = chainlink.get_price(token, block=block)
             logger.debug("stablecoin chainlink -> %s", price)
+            # If we can't get price from chainlink but `block` is after feed
+            # deploy block,feed is probably dead and coin is possibly dead.
             if price is not None:
                 return price
-        else:
-            logger.debug("stablecoin -> %s", 1)
-            return 1
+        # TODO Code better handling for stablecoin pricing
+        logger.debug("stablecoin -> %s", 1)
+        return 1
```

Comment:
can we remove this altogether and let the other markets determine this price further below if the cl feed couldn't resolve the stable ?

Reply:
I agree that this makes sense in the long run but in this PR I just wanted to fall back to the logic we used before the chainlink stables PR so we could be sure everything is able to pull. Could we leave this as an open issue in case somebody has bandwith to pick it up?

Reply:
marking this resolved per discussions in dm's